[2024-06-03 12:36:57,480: WARNING: connectionpool: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /indexes]
[2024-06-03 12:49:42,358: INFO: SentenceTransformer: Load pretrained SentenceTransformer: all-mpnet-base-v2]
[2024-06-03 12:53:45,322: INFO: SentenceTransformer: Load pretrained SentenceTransformer: all-MiniLM-L6-v2]
[2024-06-03 16:56:55,425: INFO: SentenceTransformer: Load pretrained SentenceTransformer: all-mpnet-base-v2]
[2024-06-05 12:39:05,814: INFO: SentenceTransformer: Load pretrained SentenceTransformer: all-mpnet-base-v2]
[2024-07-03 07:17:30,342: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-03 07:17:30,344: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 07:17:30,345: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-07-03 07:17:35,710: INFO: config: PyTorch version 2.2.1+cu121 available.]
[2024-07-03 07:17:35,847: INFO: refusal_data_ingestion: File Already Exist!]
[2024-07-03 07:22:53,253: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-03 07:22:53,254: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 07:22:53,255: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-07-03 07:23:02,344: INFO: config: PyTorch version 2.2.1+cu121 available.]
[2024-07-03 07:23:02,571: INFO: refusal_data_ingestion: File Already Exist!]
[2024-07-03 07:36:32,451: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-03 07:36:32,452: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 07:36:32,453: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-07-03 07:36:33,130: INFO: config: PyTorch version 2.2.1+cu121 available.]
[2024-07-03 07:36:33,357: INFO: refusal_data_ingestion: File Already Exist!]
[2024-07-03 07:45:48,167: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-03 07:45:48,168: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 07:45:48,169: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-07-03 07:45:48,693: INFO: config: PyTorch version 2.2.1+cu121 available.]
[2024-07-03 07:45:48,927: INFO: refusal_data_ingestion: File Already Exist!]
[2024-07-03 07:46:30,991: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-03 07:46:30,993: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 07:46:30,993: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-07-03 07:46:32,243: INFO: config: PyTorch version 2.2.1+cu121 available.]
[2024-07-03 07:46:32,473: INFO: refusal_data_ingestion: File Already Exist!]
[2024-07-03 07:47:31,578: INFO: common: yaml file: config/config.yaml loaded successfully]
[2024-07-03 07:47:31,579: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 07:47:31,581: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-07-03 07:47:32,178: INFO: config: PyTorch version 2.2.1+cu121 available.]
[2024-07-03 07:47:32,406: INFO: refusal_data_ingestion: File Already Exist!]
